### PR TITLE
Up to N FeederInfos

### LINKED ISSUE #303: Up to N FeederInfos (arrows)
* **Do you want to request a *feature* or report a *bug*?**
Feature

* **What is the current behavior?**
Only 2 FeederInfos expected

* **What is the expected behavior?**
Add more that 2 FeederInfos

* **What is the motivation / use case for changing the behavior?**
Some users need to add up to 5 arrows (to display 5 different values)

* **Other information**
Maybe everything's already working thanks to #284, except the space reserved to draw the arrows. Indeed, the cell height calculation shoud take into account the number of arrows. It is up to now done by `PositionVoltageLevelLayout::getFeederSpan`, which calls `LayoutParameters::getMinSpaceForFeederArrows`. We could just change that minSpaceForFeederArrows to display more arrows, but I think instead of a `minSpaceForFeederArrows` there should be a `nbMaxFeederInfos` parameter, so that the user only set this instead of number of pixel, which is less user-friendly.
Note that calling the LabelProvider to know how many arrows will be displayed when calculating the cell height is not unfortunately possible, as this provider was meant only for SVG drawing and is therefore not available during layout.

### PR DESCRIPTION
**Pull request by tadam50**
Signed-off-by: Thomas ADAM <tadam@silicom.fr>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
Fixes #303 

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
Only 2 FeederInfos expected

**What is the new behavior (if this is a feature change)?**
Add more that 2 FeederInfos

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java [hunk 1]
```diff
@@ -785,20 +785,20 @@ protected void transformArrow(List<Point> points, ComponentSize componentSize, d
...
+            // If not enough space to have layoutParameters.getArrowDistance() at both sides of the 2 feeder infos,
+            // we compute the distance between feeder anchor and first feeder info so that the two feeder infos are centered.
+            double distFeederAnchorToFirstFeederInfoCenter =
+                distancePoints >= 2 * layoutParameters.getFeederInfosOuterMargin() + 2 * componentSize.getHeight()
+                    ? layoutParameters.getFeederInfosOuterMargin()
                     : (distancePoints - 2 * componentSize.getHeight()) / 2;
-            double x = pointA.getX() + cosAngle * (distFeederAnchorToFirstArrowCenter + shift);
-            double y = pointA.getY() + sinAngle * (distFeederAnchorToFirstArrowCenter + shift);
+            double x = pointA.getX() + cosAngle * (distFeederAnchorToFirstFeederInfoCenter + shift);
+            double y = pointA.getY() + sinAngle * (distFeederAnchorToFirstFeederInfoCenter + shift);
```
Comment (flo-dup) [inline single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java:796]: Could you delete this code? I think there's no use to keep it now that the user is responsible for giving enough space for the feederInfos

File: single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java [hunk 2]
```diff
@@ -851,15 +851,15 @@ protected void insertFeederInfos(String prefixId,
             points.add(new Point(feederNode.getDiagramCoordinates()));
         }
 
-        int iArrow = 0;
+        double shiftFeederInfo2 = 0;
```
Comment (flo-dup) [inline single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java:854]: why the 2?

**Review by flo-dup — CHANGES_REQUESTED** [review]

File: single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java (hunk 2 above)
Comment (tadam50) [reply single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java]: My mistake !

File: single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java (hunk 1 above)
Comment (tadam50) [reply single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java]: I've change as following : 
double distFeederAnchorToFirstFeederInfoCenter = layoutParameters.getFeederInfosOuterMargin()

**Review by flo-dup — APPROVED** [review]

**Pull request merged**